examples/apps/ai-collab/src/types/sharedTreeAppSchema.ts

```diff
@@ -27,7 +27,7 @@ export class SharedTreeTask extends sf.object("Task", {
 	}),
 	priority: sf.required(sf.string, {
 		metadata: {
-			description: `The priority of the task in three levels, "low", "medium", "high"`,
+			description: `The case-sensitive priority of the task which which can ONLY of three levels: "Low", "Medium", "High"`,
```

Comment:
Nit:
The case-sensitive priority of the task can ONLY be one of three levels: "Low," "Medium," or "High."